src/Components/teleop-page/Stage.tsx

```diff
@@ -16,92 +16,56 @@ function Stage() {
 	const dispatch = useAppDispatch();
 	const stageValue = useAppSelector(state => state.teleop.stageClimb);
 
-	const [none, setNone] = useState<'outlined' | 'contained'>('outlined');
-	const [parked, setParked] = useState<'outlined' | 'contained'>('outlined');
-	const [single, setSingle] = useState<'outlined' | 'contained'>('outlined');
-	const [double, setDouble] = useState<'outlined' | 'contained'>('outlined');
-	const [triple, setTriple] = useState<'outlined' | 'contained'>('outlined');
+	const getButtonStyle = (status) => {
+		if (stageValue === stageValues[status]) {
+			return 'contained';
+		}
+			return 'outlined';
```

Comment:
This appears indented one too many times

Reply:
Fixed with [49f37ff](https://github.com/Team2338/GearScout2024/pull/4/commits/49f37ffe19aabe3d561ecc34df59aa6e7abdfd30)